.github/workflows/is-test.yml

```diff
@@ -0,0 +1,41 @@
+name: Integrated Service test
+
+on:
+  push:
+    branches:
+      - master
+      - integrated-service-testing
+  pull_request:
+
+jobs:
+  is-test:
+    runs-on: ubuntu-latest
+    env:
+      VERBOSE: 1
+      GOFLAGS: -mod=readonly
+
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Go build cache
```

Comment:
Haven't you concluded that it's faster without cache?

Reply:
faster without go mod cache, but not without build cache